source/polyvec/curve-tracer/spline.cpp

```diff
@@ -1045,8 +1045,8 @@ void CurveSequenceFitter::classify_evolutionary_simple(
         FittingAttempt fits[3];
 
         for (int j = 0; j < 3; ++j) {
-            bool fixFront = j & 1 == 1;
-            bool fixEnd = (j >> 1) & 1 == 1;
```

Comment:
at least according to clang, 1==1 is executed first. I assume this wasn't the intention.